controllers/propagator/propagation.go

```diff
@@ -225,14 +225,11 @@ func (r *PolicyReconciler) handleDecisions(
 	failedClusters = map[string]bool{}
 
 	for _, pb := range pbList.Items {
-		subjects := pb.Subjects
-		for _, subject := range subjects {
```

Comment:
Removing this part of the code is risky since the CRD changes will only disallow new/modification API requests but it will not guarantee that existing placement bindings before the CRD is changed will conform to this.